tests/unit_tests/test_lib/test_printer_asyncio.py

```diff
@@ -0,0 +1,60 @@
+import contextlib
+import time
+from typing import Any, Iterator, List
+from unittest.mock import patch
+
+import pytest
+from wandb.sdk.lib import printer, printer_asyncio
+from wandb.sdk.lib.printer import _PrinterTerm
+
+
+class PrinterWrapper(_PrinterTerm):
+    """A test printer that captures text set with set_text."""
```

Comment:
Since `set_text` isn't a method of `_PrinterTerm`, this description is confusing. I would describe it as "Test printer that captures text set through `dynamic_text`." I would also name the class more descriptively: "MockDynamicTextPrinter".